src/c++/simulator/geometry.hpp

```diff
@@ -97,9 +97,10 @@ struct Edge : public PrimitiveGeometry
 struct PlanarGeometry : public PrimitiveGeometry 
 {
 public:
-	Vector get_normal();
-	Vector get_unitary_normal();
-	double get_volume() override;
+    Vector get_normal() const;
+    Vector get_unitary_normal() const;
+	double get_volume() const override;
```

Comment:
difference in indentation
Do we have a .clang-format file? we can define one

Reply:
I believe we don't have one. What clang-format default should I use? hehehe